resources/helm/v2.1/mesh-config/templates/telemetryv2_1.6.yaml

```diff
@@ -1,612 +0,0 @@
-{{- if and .Values.telemetry.enabled .Values.telemetry.v2.enabled }}
```

Comment:
We need to keep this, as maistra 2.0 is based on istio 1.6

Reply:
@rcernich 1.6 and 1.7 telemetry files are removed from upstream. Do you want me to copy both into the overlay folder to make sure we still have support for the older version?

Reply:
We only need 1.6, as we never had a release with 1.7.

Reply:
Perfect. I have added back 1.6 telemetry https://github.com/maistra/istio-operator/pull/737/commits/fed344593f14912727970121c194e91126cb2ba7